refactor/src/Cage.py

```diff
@@ -132,10 +135,19 @@ def update(self, cur_date: dt.datetime, step_size: int, pressure: int) -> tuple:
         # Lice coming from reservoir
         lice_from_reservoir = self.get_reservoir_lice(pressure)
 
-        if cur_date >= self.start_date:
-            # Events that only happen when cage is populated with fish
-            # (after start date)
+        if cur_date < self.start_date:
+            # Values that are not used before the start date
+            treatment_mortality = {}
+            fish_deaths_natural, fish_deaths_from_lice = 0, 0  # type: Tuple[int, int]
+            num_infection_events = 0  # type: int
+            delta_avail_dams, delta_eggs = {}, {}  # type: Tuple[GenoDistrib, GenoDistrib]
+            avail_dams_batch = None  # type: OptionalDamBatch
```

Comment:
Not needed: the return type allows for both pytype (and possibly mypy) to infer the type of avail_dams_batch.

Types are meant to be friendly and supportive, not a nuisance to carry around!

Reply:
For both this and previous comment - I went nuclear with annotation after mypy complained. Will revise and remove unneeded annotations.